adminSiteClient/EditorDebugTab.tsx

```diff
@@ -186,6 +194,143 @@ class EditorDebugTabForChart extends React.Component<{
     }
 }
 
+@observer
+class EditorDebugTabForChartView extends React.Component<{
+    editor: ChartViewEditor
+}> {
+    @action.bound copyYamlToClipboard() {
+        // Avoid modifying the original JSON object
+        // Due to mobx memoizing computed values, the JSON can be mutated.
+        const patchConfig = {
+            ...this.props.editor.patchConfig,
+        }
+        delete patchConfig.id
+        delete patchConfig.dimensions
+        delete patchConfig.version
+        delete patchConfig.isPublished
+        const chartConfigAsYaml = YAML.stringify(patchConfig)
+        // Use the Clipboard API to copy the config into the users clipboard
+        void copyToClipboard(chartConfigAsYaml)
+        notification["success"]({
+            message: "Copied YAML to clipboard",
+            description: "You can now paste this into the ETL",
+            placement: "bottomRight",
+            closeIcon: <></>,
+        })
+    }
+
+    @observable diffModalOpen = false
+
+    @action.bound onModalClose() {
+        this.diffModalOpen = false
+    }
+
+    @computed get diffModal() {
+        return (
+            <Modal
+                open={this.diffModalOpen}
+                centered
+                width="80vw"
+                onOk={this.onModalClose}
+                onCancel={this.onModalClose}
+                cancelButtonProps={{ style: { display: "none" } }}
+            >
+                <div style={{ maxHeight: "50vh", overflowY: "auto" }}>
+                    <ReactDiffViewer
+                        newValue={stringify(
+                            this.props.editor.fullConfig,
+                            null,
+                            2
+                        )}
+                        oldValue={stringify(
+                            this.props.editor.parentConfig,
```

Comment:
I think it's nicer if we remove the properties we delete for narrative views here so they don't show up in the diff (id, slug, isPublished). I made another comment that this probably warrants a helper function now.